Common/Securities/SecurityMarginModel.cs

```diff
@@ -201,5 +201,45 @@ public virtual Order GenerateMarginCallOrder(Security security, decimal netLiqui
 
             return new MarketOrder(security.Symbol, quantity, security.Time, "Margin Call", security.Type);
         }
+
```

Comment:
Please delete the now obsolete `GenerateMarginCallOrder` function above these lines.